src/auth-service/utils/create-user.js

```diff
@@ -762,46 +673,16 @@ const join = {
           "responseFromUpdateUser in update forgotten password",
           responseFromUpdateUser
         );
-        if (responseFromUpdateUser.success === true) {
-          return {
-            success: true,
-            message: responseFromUpdateUser.message,
```

Comment:
just simple refactoring to reduce on lines of code and also align all the error responses to the AirQo standard of JSON response =`{errors: {message:"error message"}}`